licenses/LICENSE.redis

```diff
@@ -0,0 +1,10 @@
+Copyright (c) 2006-2020, Salvatore Sanfilippo
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
```

Comment:
we should care about the redistribution requirements here. Do we check the compatibility with ASL v2?

Reply:
> ASL v2

Do you mean Apache License 2.0?

> compatibility

This is a typical 3-Clause BSD License which is compatible with Apache License 2.0.

* https://www.apache.org/legal/resolved.html#category-a